example/reduce/src/reduce.cpp

```diff
@@ -32,6 +32,7 @@
 // - CpuThreads
 // - CpuOmp2Blocks
 // - CpuOmp4
+// - CpuTBBBlocks
```

Comment:
Note: here I assume TBB is okay. Also this list does not have Fibers, and the names lack `Acc`.

Reply:
checked the file `alpakaConfig.hpp` of this example and found that there is no config for TBB available

Reply:
Right, and there is a comment it's not supported. Removed my change for this file, thanks for pointing out.